### PR TITLE
[WiFi] implement saved networks

### PR DESCRIPTION
**Pull request by akaWolf**

### CODE REVIEW

File: qml/plugins/wifi/NetworkDelegate.qml [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+                pageStack.push(Qt.resolvedUrl("WifiStatus.qml"),{modelData: modelData});
+            }
+        }
+
+        actions: Rectangle{
+            width: childrenRect.width+16
+            height: parent.height
+            Image{
+                id: removeNetworkButton
+                source: "../../img/trash.svg"
```
Comment (neochapay) [inline qml/plugins/wifi/NetworkDelegate.qml]: Use absolute file patch plz

File: qml/plugins/wifi/NetworkDelegate.qml [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+                    anchors.fill: parent
+                    onClicked: {
+                        modelData.remove();
+                        model.hideAllActions(-1);
+                    }
+                }
+            }
+
+            Image{
+                source: "../../img/disconnect.svg"
```
Comment (neochapay) [inline qml/plugins/wifi/NetworkDelegate.qml]: And here

File: qml/plugins/wifi/NetworkDelegate.qml [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+                visible: (modelData.state == "online" || modelData.state == "ready")
+
+                width: 64
```
Comment (neochapay) [inline qml/plugins/wifi/NetworkDelegate.qml]: Use adaptive sizes. This https://github.com/nemomobile/qtquickcontrols-nemo/blob/master/src/controls/theme.cpp#L17 will be help for you. Use Theme.someSize value

File: qml/plugins/wifi/wifi.qml [hunk 4]
```diff
@@ -88,96 +89,51 @@ Page {
...
-                    } else {
-                        return qsTr("secure");
-                    }
-                }
+        Column{
+            id: networks
+            spacing: 10
+
+            Text{
+                text: "Saved"
```
Comment (neochapay) [inline qml/plugins/wifi/wifi.qml]: qsTr ?

Comment (akaWolf) [reply qml/plugins/wifi/wifi.qml]: fix

File: qml/plugins/wifi/wifi.qml [hunk 5]
```diff
@@ -88,96 +89,51 @@ Page {
...
-                        return qsTr("secure");
-                    }
-                }
+        Column{
+            id: networks
+            spacing: 10
+
+            Text{
+                text: qsTr("Saved")
+                color: "white"
```
Comment (neochapay) [inline qml/plugins/wifi/wifi.qml]: Theme.textColor

File: qml/plugins/wifi/wifi.qml [hunk 6]
```diff
@@ -88,96 +89,51 @@ Page {
...
-                        onClicked: {
-                            modelData.requestDisconnect();
-                            networkList.hideAllActions(-1);
-                        }
-                    }
-                }
+
+            Text{
+                text: qsTr("Enabled")
+                color: "white"
```
Comment (neochapay) [inline qml/plugins/wifi/wifi.qml]: Theme.textColor

Comment (akaWolf) [reply qml/plugins/wifi/wifi.qml]: fix

File: qml/plugins/wifi/wifi.qml (hunk 5 above)
Comment (akaWolf) [reply qml/plugins/wifi/wifi.qml]: fix

File: qml/plugins/wifi/NetworkDelegate.qml (hunk 2 above)
Comment (akaWolf) [reply qml/plugins/wifi/NetworkDelegate.qml]: fix

File: qml/plugins/wifi/NetworkDelegate.qml (hunk 1 above)
Comment (akaWolf) [reply qml/plugins/wifi/NetworkDelegate.qml]: fix

File: qml/plugins/wifi/NetworkDelegate.qml (hunk 3 above)
Comment (akaWolf) [reply qml/plugins/wifi/NetworkDelegate.qml]: fix

File: qml/plugins/wifi/WifiSettings.qml [hunk 7]
```diff
@@ -70,29 +70,34 @@ Page {
...
+    SettingsColumn{
+        TextField{
             id: passphrase
             text: modelData.passphrase
         }
 
-        Button {
+        Button{
             id: connectButton
             height: 48
```
Comment (neochapay) [inline qml/plugins/wifi/WifiSettings.qml]: Theme.size ?

File: qml/plugins/wifi/WifiStatus.qml [hunk 8]
```diff
@@ -109,7 +106,6 @@ Page {
             }
         }
 
-
         Row{
             spacing: 24
```
Comment (neochapay) [inline qml/plugins/wifi/WifiStatus.qml]: Theme.size

File: qml/plugins/wifi/wifi.qml [hunk 9]
```diff
@@ -75,11 +75,12 @@ Page {
...
     }
 
-
-    ListView {
-        id: networkList
+    Flickable{
+        visible: networkingModel.powered
         width: parent.width-size.dp(40)
         height: parent.height-actionColumn.height-size.dp(80)
+        contentHeight: networks.height+50
```
Comment (neochapay) [inline qml/plugins/wifi/wifi.qml]: size.dp(50)

File: qml/plugins/wifi/wifi.qml [hunk 10]
```diff
@@ -88,96 +89,51 @@ Page {
...
-                    }
-                }
+        Column{
+            id: networks
+            spacing: 10
+
+            Text{
+                text: qsTr("Saved")
+                color: Theme.textColor
+                font.pointSize: 30
```
Comment (neochapay) [inline qml/plugins/wifi/wifi.qml]: Theme.size

File: qml/plugins/wifi/wifi.qml [hunk 11]
```diff
@@ -88,96 +89,51 @@ Page {
...
-                } else {
-                    if (security == "none") {
-                        return qsTr("open");
-                    } else {
-                        return qsTr("secure");
-                    }
-                }
+        Column{
+            id: networks
+            spacing: 10
```
Comment (neochapay) [inline qml/plugins/wifi/wifi.qml]: Theme.size

File: qml/plugins/wifi/wifi.qml [hunk 12]
```diff
@@ -88,96 +89,51 @@ Page {
...
-                            modelData.requestDisconnect();
-                            networkList.hideAllActions(-1);
-                        }
-                    }
-                }
+
+            Text{
+                text: qsTr("Enabled")
+                color: Theme.textColor
+                font.pointSize: 30
```
Comment (neochapay) [inline qml/plugins/wifi/wifi.qml]: Theme.size

File: qml/plugins/wifi/WifiStatus.qml (hunk 8 above)
Comment (akaWolf) [reply qml/plugins/wifi/WifiStatus.qml]: fix

File: qml/plugins/wifi/WifiSettings.qml (hunk 7 above)
Comment (akaWolf) [reply qml/plugins/wifi/WifiSettings.qml]: fix

**Comment by neochapay** [comment]
LGTM

**Pull request merged**